README.md

```diff
@@ -30,6 +30,14 @@ E.g.
 curl -L $TARBALL_URL | tar -C $(helm home)/plugins -xzv
 ```
 
+Post Downloading the desired release, do note you will need to move the binary in the helm plugins folder :
```

Comment:
`curl -L $TARBALL_URL | tar -C $(helm home)/plugins -xzv` is unpacking the tarsal directly in your helm plugins folder so this could be a little confusing. It should be made clear that this is an alternative (manual) installation method.